nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/TestColdStorage.java

```diff
@@ -170,8 +178,49 @@ public void shouldFailWhenRetrievingDocumentBlobWithoutColdStorageContent() {
         }
     }
 
-    protected DocumentModel createDocument(boolean addBlobContent) {
-        DocumentModel documentModel = session.createDocumentModel("/", "anyFile", "File");
+    @Test
+    public void shouldCheckAvailabilityOfColdStorageContent() {
+        List<String> documents = Arrays.asList( //
```

Comment:
Isn't this a use case for `List.of()`?

Reply:
`List.of()` is a really immutable type of list: no adding, no removing, no update ...which is not the case of `Arrays.asList` and as i perform a sort to compare:
```
documents.sort(Comparator.naturalOrder());
            assertEquals(documents, docEvents);
```
with `ListOf` it will fail because the list will change duing to the sort.